config/pickit.ini

```diff
@@ -1,31 +1,55 @@
-; 0: Item will not be picked up
-; 1: Item will be picked up
-; 2: Item will be picked up and a message will be sent to you (if message endpoint is set e.g. discord hook)
-; 3: Item will be picked up but only stashed if it is ethereal
+; format: item_name=pickit_type, (optional list of properties to require), (optional list of properties to exclude)
+
+; pickit_type:
+; 0: item will not be picked up
+; 1: item will be picked up
+; 2: item will be picked up and a message will be sent to you (if message endpoint is set e.g. discord hook)
+
+; see images in assets/item_properties for current supported properties
+; currently only supporting properties that exist prior to identification
+; default behavior of grouped properties (a, b) is "OR". You can use and(a, b) for "AND" boolean logic
+; everything following item_name= is case insensitive
+; nested logic is not supported; i.e. you can't use and(a, or(b, c))
+; magic/rare/set/uniq items do not show # of sockets prior to id but will show eth
+
+; examples:
+; "rare_blade_talons=2" will pickup and send a discord message for all rare blade talons
+; "uniq_helm_harlequin_crest=1,, ethereal" has no required properties; will discard ethereal
+; "gray_berserker_axe=1, (socketed_5, socketed_6)" requires either 5 OR 6-socketed berserker axes
+; "gray_thresher=1, and(ethereal, socketed_4)" requires ethereal AND 4-sockets
+; "gray_thresher=1, ethereal, (socketed_1, socketed_2, socketed_3)"  requires ethereal;
+;    will discard any with 1-3 sockets. This will pickup non-socketed ethereal's.
+; "gray_weapon_grand_matron_bow=1, and(3_bow_skills, enhanced_damage, socketed_4)" will pickup but stash
+;    only GMB that have 3+ bow skills AND enhanced damage modifier AND 4 sockets
+
+; regarding ethereal items:
+; if you want to keep ONLY ethereal, add ethereal after the first comma not in () like `item=1, ethereal`
+; if you want to discard ethereal, add ethereal after the second comma not in () like `item=1,, ethereal`
+; if you want both ethereal and non-ethereal items, do not use the ethereal property
 
 [items]
 ;==================================================================;
 ;<<<<<<<<<<<<<<<<<<<<<<<<<<<< MISC >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>;
 ;==================================================================;
 
-;potions
+;POTIONS
 misc_rejuvenation_potion=0
 misc_full_rejuvenation_potion=1
 misc_super_healing_potion=1
 misc_super_mana_potion=1
 misc_greater_healing_potion=0
 misc_greater_mana_potion=0
 
-;other
-misc_tp_scroll=0
+;MISC
+;misc_tp_scroll=1
```

Comment:
I vote for removing tp_scrolls and also its asset. With buying tps I don't see any usecase picking them up.